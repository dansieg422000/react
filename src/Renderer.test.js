import React from 'react';
import renderer from 'react-test-renderer';
import Registration from './components/Registration';

// Test the UI - Snapshot
test('App Snapshot test', () => {
    const component = renderer.create(<Registration />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});