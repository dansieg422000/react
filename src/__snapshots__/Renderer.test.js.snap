// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot test 1`] = `
Array [
  "0",
  <button
    onClick={[Function]}
  >
    increment
  </button>,
]
`;
